### PR TITLE
Improvements to English grammar and spellings as well as few new answers

### PR DESCRIPTION
**Pull request by caughtinflux**

### CODE REVIEW

File: plugins/smalltalk.py [hunk 1]
```diff
@@ -394,5 +394,5 @@ def st_blow_me(self, speech, langauge):
     @register("en-US",".*sing.*song.*")
     def st_sing_song(self, speech, language):
         if language == 'en-US':
-            self.say("Daisy, Daisy, give me your answer do...")
```
Comment (onetootreefor) [inline plugins/smalltalk.py:397]: alternate response

File: plugins/smalltalk.py [hunk 2]
```diff
@@ -129,7 +129,7 @@ def st_android(self, speech, language):
         if language == 'de-DE':
             self.say("Ich denke da anders.")            
         else:
-            self.say("I think differently")
+            self.say("I don't like talking about clones.")
```
Comment (onetootreefor) [inline plugins/smalltalk.py:132]: alternate response

File: plugins/smalltalk.py [hunk 3]
```diff
@@ -50,7 +50,7 @@ def st_marry_me(self, speech, language):
         if language == 'de-DE':
             self.say("Nein Danke, ich stehe auf das schwarze iPhone von Deinem Kollegen.")            
         else:
-            self.say("No thank you, I'm in love with the black iPhone from you friend.")
+            self.say("My End User Licensing Agreement does not cover marriage.")
```
Comment (onetootreefor) [inline plugins/smalltalk.py:53]: alternate response

**Comment by janrueth** [comment]
Please refactor the plugin to choose randomly from a list of answers per type of question. Closing this pull

**Pull request closed without merging**